python/lsst/analysis/tools/atools/refCatMatchPlots.py

```diff
@@ -529,3 +533,73 @@ def setDefaults(self):
 
         self.produce.plotName = "astromDiffSky_Dec"
         self.produce.zAxisLabel = "Dec$_{{target}}$ - Dec$_{{ref}}$ (marcsec)"
+
+
+class TargetRefCatDeltaVisitMetrics(TargetRefCatDelta):
+    """Calculate the AA1, AB1, and ABF1 metrics from the difference between the
+    target and reference catalog coordinates.
+    """
+
+    AB2 = Field[float](
+        doc="Distance (in mas) from the mean at which to calculate percentage of outliers", default=20
+    )
+    coordinate = ChoiceField[str](
+        doc="The coordinate direction",
+        optional=False,
+        allowed={"RA": "Right ascension", "Dec": "Declination"},
+    )
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        self.applyContext(VisitContext)
+        self.applyContext(RefMatchContext)
+
+        self.process.buildActions.astromDiff = ConvertUnits(
+            buildAction=SubtractVector, inUnit="degree", outUnit="milliarcsecond"
+        )
+
+        self.process.buildActions.mags = ConvertFluxToMag()
+        self.process.buildActions.mags.vectorKey = "psfFlux_target"
+
+        self.process.filterActions.brightStars = DownselectVector(vectorKey="astromDiff")
+        self.process.filterActions.brightStars.selector = RangeSelector(
+            vectorKey="mags", minimum=17, maximum=21.5
+        )
+
+        self.process.calculateActions.AA1 = MedianAction(vectorKey="brightStars")
+        self.process.calculateActions.AB1 = SigmaMadAction(vectorKey="brightStars")
```

Comment:
I don't think this is what AB1 is meant to be. It should be the difference between separations measured in the r-band and those measured in any other filter. This seems to be calculating only the RMS of the separations in the input visit's filter?

Reply:
Renaming AB1 to AA1_sigmaMad.